.github/workflows/visual-snapshots-and-acceptance-py2.7.yml

```diff
@@ -8,6 +8,7 @@ on:
       - master
       - releases/**
   pull_request:
+  pull_request_target:
```

Comment:
Note, I will need to follow up and remove `pull_request`, otherwise the workflow will not run at all for this PR.

In the meantime, we may have a few PRs where VS/acceptance runs twice (once per PR event).